CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ find_package(Threads REQUIRED)
 find_package(ZLIB REQUIRED)
 
 # try to find an installed boost or install locally with conan
-set(boost_VERSION "1.71")
+set(boost_VERSION "1.80")
```

Comment:
should bump this to 1.83 too to be consistent with the conanfile.txt. might as well take the chance to upgrade to latest version IMO